pkgs/top-level/perl-packages.nix

```diff
@@ -89,22 +89,30 @@ let
   ack = buildPerlPackage {
     pname = "ack";
     version = "2.28";
+
     src = fetchurl {
       url = mirror://cpan/authors/id/P/PE/PETDANCE/ack-2.28.tar.gz;
       sha256 = "16zgn96v1qkibpj5lic571zjl07y8x55v5xql3x7bvlsmgqcnvla";
     };
+
     outputs = ["out" "man"];
+
     # use gnused so that the preCheck command passes
-    buildInputs = stdenv.lib.optional stdenv.isDarwin gnused;
+    buildInputs = stdenv.lib.optionals stdenv.isDarwin [ gnused shortenPerlShebang ];
```

Comment:
`gnused` is not needed, it should be already there

Reply:
Fixed.